omero/sysadmins/windows/install-web.txt

```diff
@@ -1,6 +1,10 @@
 OMERO.web deployment
 ====================
 
+.. Warning:: The guidance below applies to OMERO **5.2**. Code fixes for
+    OMERO.web deployment on Windows have not been back-ported to the 5.1 line
+    and this guide cannot be used to deploy OMERO 5.1.x on Windows.
```

Comment:
'... to deploy OMERO.web on OMERO 5.1.x ...' or is it clear enough given the context?